policyengine_us/variables/gov/states/hi/tax/income/credits/food_excise/hi_food_excise_exemption_amount.py

```diff
@@ -0,0 +1,47 @@
+from policyengine_us.model_api import *
+
+
+class hi_food_excise_exemption_amount(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Exemption amount for the hawaii food excise credit"
+    definition_period = YEAR
+    defined_for = StateCode.HI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.credits.food_excise_tax
+
+        income = tax_unit("adjusted_gross_income", period)
+        # Count exemptions as number of people.
+        # The legal code defines exemptions as Hawaii exemptions,
+        # which normally include additional exemptions for aged and disabled people,
+        # but excludes those additional exemptions for this program.
+        exemptions = tax_unit("exemptions", period)
+        # Reduce number of exemptions by the number of minor children
+        minor_children = (
+            tax_unit("hi_food_excise_credit_minor_child_amount", period)
+            / p.minor_child.amount
```

Comment:
we might want to make a minor child count variable to avoid this. @MaxGhenis would it be reasonable to create more variables to avoid this structure?

Reply:
Yes